ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -2577,6 +2658,88 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 15 * * 3
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 5h0m0s
+  extra_refs:
+  - base_ref: main
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: aws-4.15-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-cni-6nodes
+  spec:
```

Comment:
Please add reporter_config for slack

Reply:
Do we really want this? it might be extremely noisy in Slack just due to the nature of Payload testing.

Reply:
Is there a separate PR to submit for payload testing? If so, we could remove at that time, or create a separate channel for the noisier jobs?

Reply:
Looking at the job name this was confusing - can we add a differentiator to this?

Reply:
> Is there a separate PR to submit for payload testing? If so, we could remove at that time, or create a separate channel for the noisier jobs?

Right, let us limit scope of existing channel for cpt-ci-results only, anything beyond can be redirected accordingly.

Reply:
> so user understands payload testing job vs regular job

After chatting with Joe, I understand TRT triggers the jobs that are submitted for payload testing, thus we keep them alongside the other jobs.

We didn't account for this new payload identifier in the job renames :) ... Our available options include:
```
1. periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-cni-6nodes-payload
2. periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-payload-node-density-cni-6nodes
3. periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-cni-payload-6nodes
```

Reply:
I vote for `prefix` based namings, makes it lot easier:
`2. periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-payload-node-density-cni-6nodes`